skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -262,6 +263,10 @@ In the Teams PowerShell Module version 3.0.0 or later, the format of the Assigne
 
 - In the Teams PowerShell Module version 3.0.0 or later, the output format of Policies has now changed from String to JSON type UserPolicyDefinition.
 
+- Filtering for null polices: Admins need to query for null polices by including an empty value.
```

Comment:
Instead of null policies, it would be better to call out the scenario here

Reply:
done